src/platforms/python/guides/django/index.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Django
+redirect_from:
+  - /clients/python/integrations/django/
+  - /platforms/python/django/
+description: "Learn about using Sentry with Django."
+---
+
+The Django integration adds support for the [Django Framework](https://www.djangoproject.com/).
+
+Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io), then return to this page.
+
+<Note>
+
+Sentry's Django integration enables automatic reporting of errors and exceptions as well as performance monitoring.
+
+Django applications using Channels 2.0 will be correctly instrumented using Python 3.7. Older versions of Python require installation of [aiocontextvars](https://pypi.org/project/aiocontextvars/).
+
+</Note>
+
+## Install
+
+Install `sentry-sdk` from PyPI with the `django` extra:
+
+```bash
+pip install --upgrade 'sentry-sdk[django]'
+```
+
+## Configure
+
+To configure the Sentry SDK, initialize in your `settings.py` file.
+
+```python {filename:settings.py}
+import sentry_sdk
+
+sentry_sdk.init(
+    dsn="___PUBLIC_DSN___",
+
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for performance monitoring.
+    # We recommend adjusting this value in production,
+    traces_sample_rate=1.0,
+)
+```
+
+## Verify
+
+This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:
+
+```python
+from django.urls import path
+
+def trigger_error(request):
+    division_by_zero = 1 / 0
+
+urlpatterns = [
+    path('sentry-debug/', trigger_error),
+    # ...
+]
+```
+
+Visiting this route will trigger an error that will be captured by Sentry.
```

Comment:
```suggestion
## Verify

The snippet below includes an intentional error that will be captured by Sentry when triggered. This will allow you to make sure that everything is working as soon as you set it up:

```python
from django.urls import path

def trigger_error(request):
    division_by_zero = 1 / 0

urlpatterns = [
    path('sentry-debug/', trigger_error),
    # ...
]
```

```